src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat/TryConvertTool.cs

```diff
@@ -21,7 +21,8 @@ public class TryConvertTool : ITryConvertTool
         private static readonly string[] ErrorMessages = new[]
         {
             "This project has custom imports that are not accepted by try-convert",
-            "is an unsupported project type. Not all project type guids are supported."
+            "is an unsupported project type. Not all project type guids are supported.",
```

Comment:
This pattern is ok for now but is very much culture dependent (does try-convert localize?). We should get try-convert to return error codes or some other mechanism we can use instead of checking for error messages.

Reply:
Agreed. But I think try-convert doesn't return different error codes and these messages aren't localized, so I expect this is fine for now and we can add a work item to update try-convert/upgrade-assistant to handle this better.